docs/src/content/components/drawer.mdx

```diff
@@ -7,13 +7,17 @@ import Drawer from '@pluralsight/ps-design-system-drawer'
 
 # Drawer
 
-<TableOfContents {...props} />
+<Intro>
```

Comment:
I would refrain from technical usage comments in the Intro. I think we want to train our users to see the intro as a description of what purpose a component serves and how it is differentiated from others.

These comments are still useful. I would put them in a paragraph with its own heading as-is.  Or, I would think about how to put these comments on technical options closer to the examples that show them off.